.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: "\U0001F41B Bug Report"
+about: Something isn't working as expected
+---
+
+## Bug Report
+
+Please answer these questions before submitting your issue. Thanks!
+
+1. Which component are you using?
+(arbiter/binlogctl/drainer/pump/reparo)
+
+
+2. What did you do?
+If possible, provide a recipe for reproducing the error.
+
+
+3. What did you expect to see?
+
+
+
+4. What did you see instead?
+
+
+
+5. What version of component are you using?
+(run `component -V` in terminal, component can either be arbiter/binlogctl/drainer/pump/reparo)
```

Comment:
drop `component can ....` 
provide the relate downstream type and version of drainer

Reply:
Revised.